### PR TITLE
Fixed Nuget packages missing

### PR DESCRIPTION
**Pull request by nicolasespinoza**
### Fixed Nuget packages missing
When using the NUnit & Newtonsoft packages, there were errors regarding missing files & packages preventing a successful build.

Capture

### Jira

- Issue 1034

|    DOD    | Done |    Reviewer    | Issue | Notes |
|-----------|------|----------------|-------|-------|
| Code      | ❌   | Alex Carter |   1034 |       |
| Unit Test | N/A  | N/A            |       |       |
| Merged    | ❌  |                |       |       |

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,7 +10,6 @@ tags
 *.exe
 *.userprefs
 docs/
-build/
```
Comment (matthew17754) [inline .gitignore]: We shouldn't be tracking binaries, as that is the benefit of NuGet here.

**Comment by nicolasespinoza** [comment]
@matthew17754 We should be all good! I just tested cloning the latest version of the branch and it worked perfectly out of the box.

File: exporters/BxDRobotExporter/RobotExporterTests/obj/RobotExporterTests.csproj.nuget.dgspec.json [hunk 2]
```diff
@@ -1,34 +1,34 @@
 {
   "format": 1,
   "restore": {
-    "C:\\Users\\Victo\\Documents\\Sythesis\\synthesis\\exporters\\BxDRobotExporter\\RobotExporterTests\\RobotExporterTests.csproj": {}
+    "C:\\Users\\t_espin\\Development\\nuget\\synthesis\\exporters\\BxDRobotExporter\\RobotExporterTests\\RobotExporterTests.csproj": {}
```
Comment (wangl5) [inline exporters/BxDRobotExporter/RobotExporterTests/obj/RobotExporterTests.csproj.nuget.dgspec.json]: @matthew17754 Should this file be included if it has the full path to users' repo folder?

File: .gitignore [hunk 3]
```diff
@@ -9,7 +9,7 @@ engine/unity5/Build_Data
 tags
 *.exe
 *.userprefs
-docs/
+docis/
```
Comment (wangl5) [inline .gitignore]: @nicolasespinoza Is this a typo?

File: exporters/BxDRobotExporter/RobotExporterTests/obj/RobotExporterTests.csproj.nuget.g.props [hunk 4]
```diff
@@ -5,16 +5,15 @@
     <RestoreTool Condition=" '$(RestoreTool)' == '' ">NuGet</RestoreTool>
     <ProjectAssetsFile Condition=" '$(ProjectAssetsFile)' == '' ">$(MSBuildThisFileDirectory)project.assets.json</ProjectAssetsFile>
     <NuGetPackageRoot Condition=" '$(NuGetPackageRoot)' == '' ">$(UserProfile)\.nuget\packages\</NuGetPackageRoot>
-    <NuGetPackageFolders Condition=" '$(NuGetPackageFolders)' == '' ">C:\Users\Victo\.nuget\packages\;C:\Program Files\dotnet\sdk\NuGetFallbackFolder</NuGetPackageFolders>
+    <NuGetPackageFolders Condition=" '$(NuGetPackageFolders)' == '' ">C:\Users\t_espin\.nuget\packages\;C:\Program Files\dotnet\sdk\NuGetFallbackFolder</NuGetPackageFolders>
```
Comment (wangl5) [inline exporters/BxDRobotExporter/RobotExporterTests/obj/RobotExporterTests.csproj.nuget.g.props]: Another user-specific reference

File: .gitignore (hunk 3 above)
Comment (nicolasespinoza) [reply .gitignore]: Yes haha. Let me fix that

File: exporters/BxDRobotExporter/RobotExporterTests/obj/RobotExporterTests.csproj.nuget.dgspec.json (hunk 2 above)
Comment (nicolasespinoza) [reply exporters/BxDRobotExporter/RobotExporterTests/obj/RobotExporterTests.csproj.nuget.dgspec.json]: It doesn't seem to affect building at all, but I feel like a lot if not all files in the "obj" folder aren't necessary to include in source control. I'll experiment and see what I can figure out.

File: exporters/BxDRobotExporter/RobotExporterTests/obj/RobotExporterTests.csproj.nuget.g.props (hunk 4 above)
Comment (matthew17754) [reply exporters/BxDRobotExporter/RobotExporterTests/obj/RobotExporterTests.csproj.nuget.g.props]: I believe this one is necessary. In terms of the user-specific path, it should automatically update when rebuilt on your local machine.

**Comment by nicolasespinoza** [comment]
Alright, so I played around with the obj directory and it is definitely not necessary for source control. I tested building after deleting it, and it worked fine. Can someone verify before merging?

@matthew17754 I tested with a fresh new clone each time. There might be conflicts with other setups.

**Comment by KiraCorbett** [comment]
Confirmed

**Comment by nicolasespinoza** [comment]
> So it appears a fresh clone of master builds just fine now. Can anyone else confirm?

I think it depends on what you define as "fresh"? To test this out I made a completely new folder and cloned from there, and I'm able to reproduce it every time. I'm pretty sure Alex had this issue also so it doesn't seem exclusive to my computer or anything.

**Pull request merged**